CHANGELOG.md

```diff
@@ -1,26 +1,30 @@
 # Changelog
 
+## 0.7.0
```

Comment:
Is this a breaking change?

Reply:
We had a discussion about it yesterday as well. Rand breaks for tests (which are expanded as part of a macro). So If you depend on gridiron 0.6 and use rand 0.7 it breaks. I think it's best to just say it's breaking and move on.